src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/ADRController.kt

```diff
@@ -174,6 +174,13 @@ class ADRController(private val encryption: Encryption,
         return saveAndValidate(url, FileType.ANC)
     }
 
+    @PostMapping("/datasets/{id}/releases")
```

Comment:
API design typically sticks to consistency doesn't it? i.e. `/datasets/1/releases/1` for GET and therefore `/datasets/1/releases` for POST. Otherwise we could say the same thing about `/datasets/` itself...

Reply:
Yeah, you're right. I think i prefer GET /plural and GET /singular/id as a pattern, but since we're already using `releases` for the singular GET let's stick with that.